app/services/sufia/workflow/deposited_notification.rb

```diff
@@ -1,6 +1,6 @@
 module Sufia
   module Workflow
-    class CompleteNotification < AbstractNotification
+    class DepositedNotification < AbstractNotification
```

Comment:
Don't remove `CompleteNotification` class; Instead add something along the following:

```ruby
module Sufia
  module Workflow
    class CompletedNotification < DepositedNotification
      def initialize(*args, &block)
        # Deprecation notice goes here; saying use DepositedNotification; Any other guidance.
        super
      end
    end
  end
end
```

This cleaves to the closed for modification yet open for extension principle.